tests/test_devshell.py

```diff
@@ -125,6 +125,11 @@ def test_handles_ignores_extra_fields(self):
       self.assertEqual('sometoken', creds.access_token)
 
   def test_refuses_to_save_to_well_known_file(self):
-    with _AuthReferenceServer():
-      creds = DevshellCredentials()
-      self.assertRaises(NotImplementedError, save_to_well_known_file, creds)
+    ORIGINAL_ISDIR = os.path.isdir
+    try:
+      os.path.isdir = lambda path: True
```

Comment:
I figured it was easier than trying to find a directory which really does exist.

Also it fit in with other changes where we needed a particular directory to not exist (also harder to find on a generic system, at least harder than make `os.path.isdir` return `False`)